presto-main/src/test/java/com/facebook/presto/operator/scalar/TestRegexpFunctions.java

```diff
@@ -49,7 +72,7 @@ public static Slice invalidUtf8()
     @Test
     public void testRegexpLike()
     {
-        assertFunction("REGEXP_LIKE(invalid_utf8(), invalid_utf8())", BOOLEAN, true); // can potentially cause infinite loop
+        functionAssertions.tryEvaluate("REGEXP_LIKE(invalid_utf8(), invalid_utf8())", BOOLEAN); // can potentially cause infinite loop
```

Comment:
If this change is intentional, add a comment that explains Why. Otherwise, remove it.